prompting/organic/organic_scoring_prompting.py

```diff
@@ -157,7 +154,7 @@ async def stream_miner_chunks(uid: int, chunks: AsyncGenerator):
                     elif isinstance(chunk, StreamPromptingSynapse):
                         synapse = chunk
                 except Exception as e:
-                    logger.error(f"[Organic] Error while streaming chunks: {e}")
+                    logger.exception(f"[Organic] Error while streaming chunks")
```

Comment:
this is cool. just read that this automatically adds tracebook.  Note, you get the same effect with `logger.error(<message>, stack_info=True, exc_info=True`